src/GlobalState.php

```
@@ -35,6 +35,13 @@ public static function enrichServerVars(Request $request): array
         $server['REQUEST_METHOD'] = $request->method;
         $server['HTTP_USER_AGENT'] = '';
 
+        $parts = \parse_url($uri);
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$uri'. (undefined)

(UnusedLocalVariable)